src/pages/Login/SignIn.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import SignBtn from '../../components/Sign/SignBtn';
+import SignInput from '../../components/Sign/SignInput';
+import signInData_list from '../../components/Sign/SignInData';
+import ListComponent from '../../components/Nav/ListComponent';
+import './signin.scss';
+
+class SignIn extends Component {
+  constructor() {
+    super();
+    this.state = {
+      userInfo: {},
+    };
+  }
+
+  getValue = e => {
+    const { name, value } = e.target;
+    this.setState({
+      userInfo: { ...this.state.userInfo, [name]: value },
+    });
+  };
+
+  submitUserInfo = e => {
+    fetch('http://10.58.2.122:8000/members/login', {
```

Comment:
```suggestion
    fetch(`${API}/members/login`, {
```
src/config.js 경로에 백엔드 주소 모아서 관리해주세요